pkg/controller/plan/migration.go

```diff
@@ -1107,10 +1107,26 @@ func (r *Migration) execute(vm *plan.VMStatus) (err error) {
 		}
 		step.MarkStarted()
 		step.Phase = Running
+		// only for OVA, fetch config from the conversion pod
+		if r.Source.Provider.Type() == v1beta1.Ova {
+			pod, err := r.kubevirt.GetGuestConversionPod(vm)
+			if err != nil {
+				return err
+			}
+
+			if pod.Status.Phase != core.PodSucceeded {
+				err := r.kubevirt.GetVirtV2VConvertedVMConfig(vm, pod, step)
```

Comment:
that's a bit weird to have a function called Get... that returns only an error - how about renaming it to `updateVmByConvertedConfig`?